packages/circuits/package.json

```diff
@@ -29,7 +29,8 @@
         "buildTs": "tsc",
         "copyCircom": "rm -rf ./dist/zksnarkBuild && cp -rf ./zksnarkBuild ./dist/zksnarkBuild",
         "time": "ts-node scripts/time.ts",
-        "test": "mocha -r ts-node/register test/*.test.ts --exit"
+        "test": "mocha -r ts-node/register test/*.test.ts --exit",
+        "refresh": "find ./zksnarkBuild -type f -not -name 'powersOfTau28_hez_final_18.ptau' -delete && yarn build"
```

Comment:
you can use `'*.ptau'` instead of `'powersOfTau28_hez_final_18.ptau'`
and I think it can be called `build:overwrite` or `rebuild`
since it is related to `build`